src/tokenizer/main.py

```diff
@@ -235,14 +253,18 @@ def val(t: Tok, quote_word: bool = False) -> Any:
             # Output the tokens in CSV format, one line per token
             if t.txt:
                 print(
-                    "{0},{1},{2}".format(
-                        t.kind, quote(t.txt), val(t, quote_word=True) or '""'
+                    "{0},{1},{2},{3},{4}".format(
+                        t.kind, 
+                        quote(t.txt), 
+                        val(t, quote_word=True) or '""',
+                        quote(t.original) or '""',
```

Comment:
This probably has to be `'""' if t.original is None else quote(t.original)` (since t.original can be None)